src/dawn/CodeGen/GridTools/GTCodeGen.cpp

```diff
@@ -463,14 +518,14 @@ void GTCodeGen::generateStencilClasses(
                                                        intervalDefinitions.Levels.find(level)));
       int gt_offset =
           (level != sir::Interval::End) ? offset + 1 : (offset <= 0) ? offset - 1 : offset;
-      tss << "gridtools::level<" << gt_level << ", " << gt_offset << ", 4>";
+      tss << "gridtools::level<" << gt_level << ", " << gt_offset << ", 3>";
```

Comment:
I'm not sure about this 3. That should be the offset_limit of axis, which is 2 by default. But I am not 100% sure if this is also set here somewhere. I didn't find it yet.